src/gstreamer/launch/laptop_launch.py

```diff
@@ -0,0 +1,23 @@
+from launch import LaunchDescription
+from launch_ros.actions import Node
+from launch.actions import ExecuteProcess
+
+
+# Launches the joystick node and the gstreamer camera client node on the team laptop
+def generate_launch_description():
+    ld = LaunchDescription()
+
+    joystick_node = Node(
+        package="joy",
+        executable="joy_node",
+        parameters=["config/joy_node.yaml"],
+    )
+ 
+    start_gStreamer_client = ExecuteProcess(
+        cmd=['rqt', '--force-discover'],
+        shell=True, output='screen')
```

Comment:
you could technically make this open a specific rqt view

Reply:
@jmblixt3 Good idea, how do we do that?

Reply:
done!